src/amo/components/ContributeCard/index.js

```diff
@@ -71,6 +84,14 @@ export const ContributeCardBase = ({ addon, i18n }: Props) => {
       break;
   }
 
+  const onButtonClick = (event: SyntheticEvent<HTMLAnchorElement>) => {
+    _tracking.sendEvent({
+      action: addon.slug,
+      category: CONTRIBUTE_BUTTON_CLICK_CATEGORY,
+      label: event.currentTarget.href,
```

Comment:
We usually pass the addon's name in `label`.

Reply:
See my comment below.